src/client/components/Editor.tsx

```diff
@@ -32,6 +34,37 @@ class Editor extends React.PureComponent<EditorProps> {
         }
     }
 
+  componentDidMount(): void {
+      // TODO: why does using gutterclick yield weird highlighting?
+      this.aceComponent.current.editor.on("gutterclick", function(e) {
```

Comment:
Try to use single quotes instead of double quotes, and arrow functions instead of classic functions (I think the linter complains about this)

```
this.aceComponent.current.editor.on('gutterClick', (e) => {
```

You'll also need to specify a type for `e`, although I don't know off the top of my head which type to use.